libs/ngx-mime/src/lib/viewer/viewer.component.scss

```diff
@@ -7,6 +7,12 @@
   display: flex;
   flex-direction: column;
 
+  .openseadragon {
+    user-select: none;
+    -webkit-user-select: none;
+    -moz-user-select: none;
```

Comment:
We don't need these vendor prefixes. We only target modern browsers, and the unprefixed `user-select` property is supported in all major browsers from 2020 onwards. This includes Safari 12.1 and later, which was released in 2019.